targets/kde

```diff
@@ -7,9 +7,13 @@ DESCRIPTION='Installs a minimal KDE desktop environment. (Approx. 600MB)'
 HOSTBIN='startkde'
 CHROOTBIN='crouton-noroot startkde'
 . "${TARGETSDIR:="$PWD"}/common"
-
+if [release -lt vivid]; then 
```

Comment:
This needs to be done after the `### Append to prepare.sh:` line.
There also should not be any brackets around the condition, since `release` is a function.

How does this interact with Debian and Kali?  `release -lt vivid` will return false on all Debian and Kali distros--do all of the supported ones use plasma-desktop or kde-plasma-desktop?